test/formats.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd scripts
+python alternate_bulk_formats.py
+cd -
+if [ $(git diff alternate_formats/* | wc -l) -eq 0 ]; then
+  echo "No alternate format changes needed..."
+else
+  echo "Alternate formats need regenerated.  Run alternate_bulk_formats.py"
```

Comment:
My vote would be to remove CSV from the set of supported bulk formats. We've seen strong community interest in JSON, but less so from CSV.